src/main/java/com/non/controller/UserController.java

```diff
@@ -1,14 +1,21 @@
 package com.non.controller;
 
+import com.non.exception.ApiException;
 import com.non.model.User;
 import com.non.service.UserService;
+import com.non.util.PspServiceConstants;
+import com.scaffolding.util.PlatformConstants;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.CrossOrigin;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.server.ResponseStatusException;
 
 import java.util.List;
 
 @RestController
+@CrossOrigin(origins = PspServiceConstants.ACCESS_CONTROL_ALLOW_ORIGIN_VALUE, allowedHeaders = PspServiceConstants.ACCESS_CONTROL_ALLOW_ORIGIN_VALUE)
 public class UserController {
```

Comment:
Can we please document the usage of CORS and in case we wanted to avoid setting would that have been possible?